app/src/main/java/com/marvel999/fuitemplet/FakeData.kt

```diff
@@ -0,0 +1,68 @@
+package com.marvel999.fuitemplet
+
+import androidx.annotation.DrawableRes
+
+data class FakePostData(
+    @DrawableRes val userImage: Int,
+    val userName: String,
+    val likes: Int,
+    val postDescription: String,
+    val postTime: String,
+    @DrawableRes val postImage: Int,
+)
+
+val usersPosts = listOf(
+
+    FakePostData(
+        userImage = R.drawable.model_s,
```

Comment:
Instead of adding image in app can you use https://unsplash.com/

Reply:
It's is an online process right?

Reply:
right of let's leave for now